src/braket/circuits/quantum_algorithm.py

```diff
@@ -0,0 +1,76 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from __future__ import annotations
+
+from typing import List
+
+from braket.circuits.quantum_operator import QuantumOperator
+
+
+class QuantumAlgorithm(QuantumOperator):
```

Comment:
I'm not sure if `QuantumAlgorithm` is what this represents. The docstring seems to indicate that this behaves more like a subcircuit.

Reply:
That is a typo. It should read as follows:
```
An algorithm that can be distinctly represented in a circuit.
```